apps/DemoApp/DemoApp/ViewController.m

```diff
@@ -1,42 +1,60 @@
 #import "ViewController.h"
 
 @import MUXSDKStats;
+@import Mux_Stats_Google_IMA;
 
 static NSString *DEMO_PLAYER_NAME = @"demoplayer";
 
-@interface ViewController ()
+@interface ViewController () {
+    AVPlayer *_avplayer;
+    AVPlayerViewController *_avplayerController;
+    NSTimer *_videoChangeTimer;
 
+    // IMA SDK variables
+    IMAAdsLoader *_adsLoader;
+    IMAAdsManager *_adsManager;
+    IMAAVPlayerContentPlayhead *_contentPlayhead;
+    MuxImaListener *_imaListener;
+    MUXSDKPlayerBinding *_playerBinding;
+}
 @end
 
 @implementation ViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
     _avplayerController = [AVPlayerViewController new];
-//    AVPlayer *player = [self testImaSDK];
+    AVPlayer *player = [self testImaSDK];
 //    AVPlayer *player = [self testAVQueuePlayer];
-    AVPlayer *player = [self testAVPlayer];
+//    AVPlayer *player = [self testAVPlayer];
     [self setupAVPlayerViewController: player];
 }
 
+- (void) viewDidAppear:(BOOL)animated {
+    [self requestAds];
```

Comment:
This is now required to be called from `viewDidAppear` because ads must be requested after the view showing the ads is in the view hierarchy